regtests/polaris-reg-test

```diff
@@ -29,22 +23,16 @@ if [ ! -d ${SCRIPT_DIR}/polaris-venv ]; then
   pip install poetry==1.5.0
 
   cp ${SCRIPT_DIR}/regtests/client/python/pyproject.toml ${SCRIPT_DIR}
-
-  # Save the current directory
-  CURRENT_DIR=$(pwd)
-  cd $SCRIPT_DIR && poetry install
-  cd $CURRENT_DIR
+  pushd $SCRIPT_DIR && poetry install ; popd
 
   deactivate
   echo "First time setup complete."
 fi
 
-# Save the current directory
-CURRENT_DIR=$(pwd)
-cd $SCRIPT_DIR > /dev/null
+pushd $SCRIPT_DIR > /dev/null
```

Comment:
`pushd` is not supported in all shells, which is why this script originally didn't use it

Reply:
Yeah, that is what I was guessing as well. But I did check last night that spark python image does support it. Maybe I am missing something here?

Reply:
I think it depends on the shell docker uses. Maybe it's better to avoid `pushd` etc. if we can

Reply:
We should use the variable in both places if `pushd` isn't supported in all shell. Thoughts?

Reply:
Maybe, although the `polaris` script is meant to be run in a different environment and we can reasonably expect that the shell header will be respected. That's not necessarily true for a docker container used for regression tests. 

In any event changing this is probably outside the scope of this PR. I'd like to get this fix merged ASAP as main is broken.

Reply:
Let's merge the previous commit then. @MonkeyCanCode , can you revert the latest one?

Reply:
@flyrain / @eric-maynard changes are reverted (diff file content and diff name). I am not aware of the main is broken issue and this change won't fix that I think?

Reply:
As you noted in the description, the CLI's first-time setup current doesn't work and so the CLI is unusable without the user doing a workaround.